src/relay/src/hooks/__tests__/useMutation.test.js

```diff
@@ -0,0 +1,148 @@
+// @flow
+
+/* global document */
+
+import React, { useEffect } from 'react';
```

Comment:
AFAIK we always imported everything `import * as React from 'react';`

Reply:
There is nothing wrong with this import

Reply:
That’s true, I’m trying to point out consistency

Reply:
Feel free to create an Eslint rule if you think it's essential. I think it's better to focus on other things than consistency, which in this case, doesn't bring anything.